docker/web/production/crontab

```diff
@@ -0,0 +1,12 @@
+# ┌───────────── minute (0 - 59)
+# │ ┌───────────── hour (0 - 23)
+# │ │ ┌───────────── day of month (1 - 31)
+# │ │ │ ┌───────────── month (1 - 12)
+# │ │ │ │ ┌───────────── day of week (0 - 6) (Sunday to Saturday;
+# │ │ │ │ │                                       7 is also Sunday on some systems)
+# │ │ │ │ │
+# │ │ │ │ │
+# * * * * *  command to execute
+# Don't remove the empty line at the end of this file. It is required to run the cron job
+0 3 * * * ./web photo weeklyphoto >/dev/null 2>&1
+0 2 * * * apt-get update && apt-get upgrade -y
```

Comment:
Are you sure you want to do this? Seems like a good way to break your container at 2 am.

Reply:
Well, if you don't want to build images all the time to keep the container up to date, it seems like a good idea. I'm not sure if this is the best solution though.

Reply:
I think the code of the website gets updated frequently enough to cover all essential updates. Everytime you rebuild the container you also get all the updates.

Reply:
That's true. But there are times we do not merge PRs into master for months.

Reply:
Which is probably fine since there usually aren't any important updates. And if there is a very critical (security) update you can rebuild it manually.

Reply:
Alright, I will remove it